src/System.Management.Automation/namespaces/ProviderBase.cs

```diff
@@ -15,10 +15,12 @@
 
 namespace System.Management.Automation.Provider
 {
+
     /// <summary>
     /// This interface needs to be implemented by providers that want users to see
     /// provider-specific help.
     /// </summary>
+#nullable enable
```

Comment:
💭 This seems a bit strange. I would expect to see it before the documentation comment.